Project.toml

```diff
@@ -0,0 +1,16 @@
+name = "CLFFT"
+uuid = "fe1fc2a7-b730-4eae-aba2-587af0173dc9"
+authors = []
```

Comment:
The authors of this package should be listed here. Please double check the original creators and contributors in the license.